app/controllers/reviews_controller.rb

```diff
@@ -8,13 +8,18 @@ class ReviewsController < ApplicationController
   def create
     version = Work.find(params[:work_id]).head
     authorize! version, to: :review?
+    create_event(version)
+
+    redirect_to dashboard_path
+  end
+
+  def create_event(version)
```

Comment:
I'm not in love with this name because creating an "Event" is one of two things that happens here (and it's the less important thing).  The other is that it causes a transition.  How about `record_transition` instead?

Reply:
Oh, you're right. That is a terrible method name. Fixing.